eventhubs-capture-synapse-delta/create-solution.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+# Strict mode, fail on any error
+set -euo pipefail
+source set-variables
+source ../streaming/synapse/job/run-synapse-pipeline.sh
+
+on_error() {
+    set +e
+    echo "There was an error, execution halted" >&2
+    echo "Error at line $1"
+    exit 1
+}
+
+trap 'on_error $LINENO' ERR
+
+export SQL_ADMIN_USER="sasesssynadmin"
```

Comment:
why these strange values? 

In create-sql.sh we hard-code the admin user (for simplicity) to the value `serveradmin`. Is there a reason to do things differently here?